be/src/agent/task_worker_pool.cpp

```diff
@@ -1150,6 +1150,9 @@ void TaskWorkerPool::_report_disk_state_worker_thread_callback() {
             disk.__set_disk_total_capacity(root_path_info.disk_capacity);
             disk.__set_data_used_capacity(root_path_info.data_used_capacity);
             disk.__set_disk_available_capacity(root_path_info.available);
+            std::filesystem::path trash_path(root_path_info.path + TRASH_PREFIX);
+            disk.__set_trash_used_capacity(
```

Comment:
Why not update the trash dir size in `DataDir::update_capacity()`, just like `data_used_capacity`?

Reply:
Now I moved it to StorageEngine::get_all_data_dir_info. In fact, DataDir::update_capacity() only updates disk_capacity and available, while data_used_capacity and trash_used_capacity have different update methods.